### PR TITLE
update dacfx to 18.6

### PR DESCRIPTION
**Pull request by dzsquared**
# Description
Updating DacFx from 18.5.1 to 18.6

## Check list
- [X] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [X] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by HAWAIIBEATCH — APPROVED** [review]

**Comment by HAWAIIBEATCH** [comment]
readme-edits

**Review by al-cheb — APPROVED** [review]

**Comment by miketimofeev** [comment]
@dzsquared could you please add DacFx to the software report(probably to the tools section)? Please feel free to ping us if you need any help with that.

**Comment by dzsquared** [comment]
@miketimofeev I gave adding DacFx to the report a shot and added a working function for returning the dacfx version.  Not sure it's well written, however.

File: images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1 [hunk 1]
```diff
@@ -260,3 +260,9 @@ function Get-AZDSVersion {
     $azdsVersion = $(azds --version) | Select-String "(\d+\.\d+\.\d+.\d+)"
     return "Azure Dev Spaces CLI $azdsVersion"
 }
+
+function Get-DacFxVersion {
```
Comment (maxim-lobanov) [inline images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1:264]: Could you please also add call of this function to https://github.com/actions/virtual-environments/blob/main/images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

**Review by maxim-lobanov — APPROVED** [review]

**Comment by miketimofeev** [comment]
@dzsquared sorry for that, but we introduced new DB tools section in the scope of this PR https://github.com/actions/virtual-environments/pull/1706 Could you please pull the latest master and move (Get-DacFxVersion) in SoftwareReport.Generator.ps1 under this new section?

**Comment by dzsquared** [comment]
> @dzsquared Drew Skwiers-Koballa FTE sorry for that, but we introduced a new **Database tools** section in the scope of this PR #1706 Could you please pull the latest master and move (Get-DacFxVersion) in SoftwareReport.Generator.ps1 under this new section?

this should be complete

**Review by miketimofeev — APPROVED** [review]

**Review by alepauly — APPROVED** [review]
thank you @dzsquared!

**Comment by maxim-lobanov** [comment]
@dzsquared , Win2016 validation has failed due to known issue that was fixed recently in #1751 . Could you please pull the latest `main` to rerun checks. Sorry for inconvenience.

**Pull request merged**